docs/user-guide/advanced_topics/networking.md

```diff
@@ -0,0 +1,43 @@
+
+## `broker_address`, `broker_port`, `local_interface` and `local_port`
+
+Regardless of which core you're using, there are a few specific networking options that allow for changes to default values to enable working in a more restrictive networking environment.
+
+- **`broker_address`** and **`broker_port`** - for sub-brokers or federates, defines the IP address (`broker_address`) and port (`broker_port`) which should be used to connect to a parent broker
+- **`local_interface`** and **`local_port`** - defines the IP address (`local_interface`) and port (`local_port`) where a broker, sub-broker, or federate will look for connections to the federation.
+
+The `broker_port` is typically defined as a command line switch when instantiating the broker. `broker_address` and `broker_port` are also used as options when configuring federates to define the socket they should connect to. These options can be set as part of the `fed_init_string`, `core_init_string` or part of the JSON configuration for that federate. The `local_port` options can be similarly configured. Further details on these configuration methods can be found in the [Configuration Options Reference](../../references/configuration_options_reference).
+
+It is also possible to include the port number when defining `broker_address` (effectively defining the broker's socket) in addition to the IP address such as in the format: `192.169.0.1:23400`. Doing so would then not require the `broker_port` option to be defined.
+
+As of this writing there is a generic `port` option supported in HELICS that tries to be all things to all people. Experience has shown that though well-intentioned, the feature of it being generic has become a bug in that it causes confusion among users. You may still see it lurking in examples or documentation but it is recommended that its use be avoided and the more explicit `broker_port` and `local_port` be used instead.
+
+## Broker Hierarchies and Multi-Computer Federations
+
+Once a federation reaches a certain size, it is not unusual for it to end up deployed across multiple compute nodes and often this results in establishing a broker hierarchy to reduce traffic between compute nodes (and help the co-simulation to run faster). In complex networking environments, this will likely entail the use of the `tcp_ss` core and the specification of the broker and sub-broker sockets.
+
+The good news is that we already have two other pages of documentation devoted to this and both include running examples that show how these features can be put to use. Here's the [documentation on broker hierarchies](./broker_hierarchies) and here's the one on [running across multiple compute nodes](./borker_multicomputer).
+
+Nadia's stuff:
```

Comment:
I would remove this line. A reader of the docs will not know what this is referring too.